core/tests/protractor_desktop/userJourneys.js

```diff
@@ -59,6 +62,8 @@ describe('Basic user journeys', function() {
       await general.checkForConsoleErrors([]);
 
       await browser.get(general.MODERATOR_URL_SUFFIX);
+      await waitFor.visibilityOf(
+        errorContainer, '401 Error page taking too long to appear.');
```

Comment:
Ned to validate whether it's 401 or 404 or 500, right?

Reply:
done